includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -162,6 +164,19 @@ public static function add_admin_hooks() {
 		} );
 	}
 
+	/**
+	 * Enqueue style.
+	 */
+	public static function enqueue_admin_assets() {
+		// Styles.
+		wp_enqueue_style(
```

Comment:
This should only be enqueued for the post list screen for this specific post type